components/button/Button.module.css

```diff
@@ -1,8 +1,17 @@
 .btn {
-  border-radius: 10px;
+  display: grid;
+  place-items: center;
+  padding: 10px;
```

Comment:
Is there a specific reason why you used pixel instead of rem/em here? Otherwise i would suggest using rem/em instead of pixel.

Reply:
No, I can't give you a good reason for that. I'm going to change it to em. Thanks :)